File: app/src/main/java/com/oluwatimilehin/cryptoconverter/data/ExchangeRate.java
package com.oluwatimilehin.cryptoconverter.data;

import com.google.gson.annotations.SerializedName;

import java.util.HashMap;

/**
 * Created by Oluwatimilehin on 12/10/2017.
 * oluwatimilehinadeniran@gmail.com.
 */

public class ExchangeRate {

    @SerializedName("ETH")
    private HashMap<String, Double> ETHRates;

    public HashMap<String, Double> getETHRates() {
        return ETHRates;
    }

    @SerializedName("BTC")
    private HashMap<String, Double> BTCRates;

    public HashMap<String, Double> getBTCRates() {
        return ETHRates;
    }

}

File: app/src/main/java/com/oluwatimilehin/cryptoconverter/data/ExchangeRate.kt
package com.oluwatimilehin.cryptoconverter.data

import com.google.gson.annotations.SerializedName

/**
 * Created by Oluwatimilehin on 19/10/2017.
 * oluwatimilehinadeniran@gmail.com.
 */
data class ExchangeRate(
        @SerializedName("ETH")
        val ethRates: HashMap<String, Double>,

        @SerializedName("BTC")
        val btcRates: HashMap<String, Double>
)

